src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/EnumGroupAndName.cs

```diff
@@ -28,7 +30,28 @@ public EnumGroupAndName(string group, string name)
             }
 
             Group = group;
-            Name = name;
+            _nameFunc = () => name;
```

Comment:
That would change the behavior because passing a null `name` would turn into a function that returns null and then not throw, right? It's a small difference, but it doesn't really seem worth it to me for some internal cleanup.